price_feeds_poller.py

```diff
@@ -98,18 +99,25 @@ def handle_requestUpdate(
       print(f"   xx Transaction rejected: {ex}")
       return [ 0 ]
 
+    #print(f" > Tx. receipt   :", receipt)
+    
     # Check if transaction was succesful
     if receipt['status'] == False:
       print(f"   $$ Transaction reverted !!")
       return [ -1, tx.hex() ]
-    else:      
-      logs = contract.events.PriceFeeding().processReceipt(receipt, errors=DISCARD)
+    else:
       requestId = 0
-      if len(logs) > 0:
+      if len(receipt['logs']) > 0:
+        logs = contract.events.PriceFeeding().processReceipt(receipt, errors=DISCARD)
         requestId = logs[0].args.queryId
-      if requestId > 0:
-        print(f" <<<< Request id : {requestId}")      
-      return [ requestId, tx.hex(), total_fee ]
+        if requestId > 0:
+          print(f" <<<< Request id : {requestId}")
+        else:
+          print(f" <<<< Synchronous update.")
```

Comment:
What's a synchronous update?

Reply:
WitnetPriceFeedRoutedCached price feeds get re-calculated (i.e. updated) upon call to `requestUpdate()`, so no DR is actually posted to the WRB, an no need to wait to a finalized response from Witnet.